packages/sdk-js/package.json

```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "mocha": "^8.2.0",
+    "openapi3-ts": "^2.0.0",
```

Comment:
No need to add it to devDependencies here. It's pulled in from the root package.json (monorepo's ftw)

Reply:
saad74657@gmail.com

Reply:
Awesome!

When the user installs standalone sdk, he/she won't have `openapi3-ts` in `node_modules`, or am I missing something?
I think we either need to either add it to `dependencies`, embed `OpenApiObject` in project or simply return `any`?

This is `server.d.ts`
```typescript
import { AxiosInstance } from 'axios';
import type { OpenAPIObject } from 'openapi3-ts';
export declare type ServerInfo = {
    project: {
        project_name: string;
        project_logo: string | null;
        project_color: string | null;
        public_foreground: string | null;
        public_background: string | null;
        public_note: string | null;
        custom_css: null;
    };
    directus?: {
        version: string;
    };
    node?: {
        uptime: number;
        version: string;
    };
    os?: {
        totalmem: number;
        type: string;
        uptime: number;
        version: string;
    };
};
export declare class ServerHandler {
    private axios;
    constructor(axios: AxiosInstance);
    specs: {
        oas: () => Promise<OpenAPIObject>;
    };
    ping(): Promise<'pong'>;
    info(): Promise<{
        data: ServerInfo;
    }>;
}
```